spec/requests/sidebar_sections_controller_spec.rb

```diff
@@ -238,6 +241,48 @@
 
       expect(sidebar_url_3.reload.name).to eq("other_tags")
     end
+
+    it "doesn't allow users to edit community section" do
+      sign_in(user)
+      put "/sidebar_sections/#{community_section.id}.json",
+          params: {
+            title: "custom section edited",
+            links: [],
+          }
+
+      expect(response.status).to eq(403)
+    end
+
+    it "allows admin to edit community section" do
+      sign_in(admin)
+      everything_link = community_section.sidebar_urls.find_by(name: "Everything")
+      my_posts_link = community_section.sidebar_urls.find_by(name: "My Posts")
+      community_section
+        .sidebar_section_links
+        .where.not(linkable_id: [everything_link.id, my_posts_link.id])
+        .destroy_all
+      put "/sidebar_sections/#{community_section.id}.json",
+          params: {
+            title: "community section edited",
+            links: [
+              { icon: "link", id: my_posts_link.id, name: "my posts edited", value: "/my_posts" },
+              {
+                icon: "link",
+                id: everything_link.id,
+                name: "everything edited",
+                value: "/everything",
+              },
+            ],
+          }
+
+      expect(response.status).to eq(200)
+
+      expect(community_section.reload.title).to eq("community section edited")
+      expect(community_section.sidebar_urls[0].reload.name).to eq("my posts edited")
+      expect(community_section.sidebar_urls[0].value).to eq("/my_posts")
+      expect(community_section.sidebar_urls[1].reload.name).to eq("everything edited")
+      expect(community_section.sidebar_urls[1].value).to eq("/everything")
```

Comment:
```suggestion
      expect(community_section.sidebar_urls[0].name).to eq("my posts edited")
      expect(community_section.sidebar_urls[0].value).to eq("/my_posts")
      expect(community_section.sidebar_urls[1].name).to eq("everything edited")
      expect(community_section.sidebar_urls[1].value).to eq("/everything")
```

The reload is not necessary here since reloading the `community_section` will already cause the associations to be loaded from the DB again.